onnxruntime/core/providers/cuda/cu_inc/common.cuh

```diff
@@ -208,8 +208,9 @@ static INT CeilDiv(INT a, INT2 b)  // ceil(a/b)
 
 struct GridDim {
   enum : CUDA_LONG {
-    maxThreadsPerBlock = 1024,  // use this many threads per block
-    maxWarpsPerBlock = 32,      // use this many warps per block. This means 1024 threads for warpSize=32
+    maxThreadsPerBlock = 256,  // max threads per block
```

Comment:
I think a lot of non-elementwise operators are using this constant as well. is there a potential regression for those operators?

Reply:
Between 128 and 256 threads per block is a better choice in general. It can saturate the SM and distribute work load more equally to SMs: https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#thread-and-block-heuristics